app/src/main/java/com/plcoding/calorytracker/MainActivity.kt

```diff
@@ -45,12 +44,13 @@ class MainActivity : ComponentActivity() {
                 Scaffold(
                     modifier = Modifier.fillMaxSize(),
                     scaffoldState = scaffoldState,
-                ) {
+                ) { padding ->
```

Comment:
> Since Compose 1.2.0 it's required to use padding parameter, passed into Scaffold content composable. You should apply it to the topmost container/view in content

https://stackoverflow.com/questions/72084865/content-padding-parameter-it-is-not-used